lib/moon_web/examples/design/bottom_sheet_example/responsive.ex

```diff
@@ -0,0 +1,92 @@
+defmodule MoonWeb.Examples.Design.BottomSheetExample.Responsive do
+  @moduledoc false
+
+  use Moon.StatefulComponent
+  use MoonWeb, :example
+
+  alias Moon.Design.Button
+  alias Moon.Design.BottomSheet
+
+  def render(assigns) do
+    ~F"""
+    <div>
+      <Button on_click="set_open">
+        Modal
+      </Button>
+      <BottomSheet id="modal_bottom_sheet" class="lg:hidden lg:flex lg:items-center lg:justify-center">
```

Comment:
How this is 0-css?

Reply:
really not sure if i've got the point of this comment

Reply:
not sure if this comment has any point

Reply:
The point of Moon is, that application developer can use them without writing CSS.

In this example, BottomSheet is displayed as Modal on bigger devices. Functionally it works. But:

1. If Figma has default behaviour for Modal-BottomSheet then why it does not work automatically like that? Could there be some helper classes? Why Application Developer needs to know about classes for Modal or BottomSheet?

2. There are many other scenarios, where default behaviour of Moon needs to match with Figma? Applying same way, then changing behaviour or visuals, this "hacky way" would beat the purpose of Moon Design System (not component library) partly?

Reply:
answering, that still looks like more quarrel than a productive discussion
point of Moon - agree, can be used, but cannot be customised
1. can't see Modal-BottomSheet example in figma, can see Responsive Popover & Dropdown there
2. again - not sure got the question. 

main point of this PR is to show that Moon components can be +/- easy customised to achieve almost any modifications using existing components. I'm pretty sure I haven't in mind the ability to on-a-fly-replace Modal with BottomSheet (or any other components)  when developing, but really happy to see it's possible. (and easy)

for the flexibility reasons right now some front-end knowledge is required to implement these modifications. although tailwind-css is fun and interesting, i'm not going to make backend developers to learn it. I'm pretty sure we can extend this approach with some helpers or attributes (less flexible way, but 0-css). 

Moon strategy is to start developing from design team, and right now in figma we have only few points (btw, not a Modal-BottomSheet, but for Dropdown-BottomSheet and Popover-Drawer, not sure about the reasons).

Reply:
@phgrey Very good! What is ETA for 0-css behaviour?

Reply:
directly here i'll need ~2h to implement some stable API. solution will still be a bit ugly inside, but it will work and API will be stable. (meaning no breaking changes later, all changes inside only)

but i really want some approval from our decision-makers - @Egert1425, @dkireev